src/static/js/common_html.js

```diff
@@ -321,19 +321,16 @@ const html = {
             if (a.FLVRESULT == 2) { p.push(_("FLV+")); }
             s.push(html.icon("positivetest", p.join(" ")));
         }
-        if (config.bool("EmblemRabies") && !a.RABIESTAG && 
-            config.str("AlertSpeciesRabies").split(",").indexOf(String(a.SPECIESID)) != -1) {
+        if (config.bool("EmblemRabies") && !a.RABIESTAG) {
             s.push(html.icon("rabies", _("Rabies not given")));
         }
         if (config.bool("EmblemSpecialNeeds") && a.HASSPECIALNEEDS == 1) {
             s.push(html.icon("health", _("Special Needs")));
         }
-        if (config.bool("EmblemUnneutered") && a.NEUTERED == 0 && 
-            config.str("AlertSpeciesNeuter").split(",").indexOf(String(a.SPECIESID)) != -1) {
+        if (config.bool("EmblemUnneutered") && a.NEUTERED == 0) {
```

Comment:
Same here, I might be misunderstanding, but this is going to show the not neutered emblem for all species of animals now?